specification/patient-demographics-service-api.yaml

```diff
@@ -121,85 +118,23 @@ info:
 of birth (range)\n\n## Historic data search\nBy default, the endpoint only searches current information. Use the `_history` query parameter to search historic information, such as maiden names or previous addresses.  \n\n## Matching rules\nSearch parameters are not case sensitive.\n\nSpaces in postcodes must match exactly, for example 'AA11 1AA' will not match 'AA111AA'.\n\nFor date of birth and date of death, you can search for an exact date or for a range of dates.\n\nYou can use wildcards in given name and family name. Wildcards must be preceded by at least two characters. For example 'Br*' is valid, but '*cDougal' is not. Multiple wildcards are allowed; for example 'Br*w*' will match 'Brown' and 'Brower'.\n\nYou can also request a fuzzy search ( `_fuzzy-match` parameter). This search does the best it can to find a match by:\n  * matching common homophones, such as 'Smith' and 'Smythe'\n  * checking for transposed names, such as 'Adam Thomas' and 'Thomas Adam'\n  * doing phonetic matching using soundex\n\n## Result scoring\nEvery result includes a score from 0 to 1. Scores less than one can happen when supplied parameters do not exactly match the values on the patient record.\nResults are sorted best-first.\nUse the `_exact-match` parameter to return only exact matches (score = 1).\n\n## Number of results returned\nBy default, the endpoint will return a maximum of 50 results. If there are more than 50 matching patient records, the endpoint will return both no results and the error response TOO_MANY_MATCHES.\n\nUse the `_max-results` parameter to limit the number of results returned.\n\n## Clinical safety and IG\nTBC guidance around safe and private use of search.\n\n\n## Sandbox testing\nTest the following scenarios in our [sandbox environment](/testing#sandbox-testing):\n\n| Scenario                       | Request                                                      | Response                                   |\n| ------------------------------ | ------------------------------------------------------------ | ------------------------------------------ |\n| Successful simple search       | `family`=`Smith`, `given`=`Jane`, `gender`=`female`, `birthdate`=`2010-10-22` | HTTP Status 200 with single search result |\n| Unsuccessful search            | Any other valid search query parameters                      | HTTP Status 200 with no search results     |\n| Invalid date format            | `birthdate` or `death-date` query parameters with values not in format `YYYY-MM-DD` | HTTP Status 400 with problem description   |\n| Too few search parameters      | Any invalid combination of query parameters                  | HTTP Status 400 with problem description   |\n\nYou can try out the sandbox using the 'Try this API' feature on the right-hand side of this page.\n\nAlternatively, try out the sandbox using our Postman collection:\n[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/2871d49488699f3f5a67)\n"
```

Comment:
This must have been something stoplight did. It's not a problem but it does make it less readable from editing raw text.